### PR TITLE
chore(website): Add MVP Videos area to Success Stories Page

### PR DESCRIPTION
**Pull request by dnilasor**
I'd like to add an area to the new Success Stories page to house compelling video user stories from the Spinnaker community. The value add is to show some raw footage of user testimonials that highlight the awesomeness and ubiquity of Spinnaker, on the same site that helps users get started with the platform. We have a growing volume of video testimonials and it'd be great to leverage them here. Nothing better than hearing real users talk about the impact of Spinnaker on their sw delivery, what migration was like, how developers at their orgs viewed adoption, the ease and importance of guardrails, the value of an OSS solution vs proprietary, etc.

A previous attempt at this by Drodio (https://github.com/spinnaker/spinnaker.github.io/pull/1073) was not merged after concern was raised about the balance of Armory branding in the included videos. I brought this up at this week's Docs SIG meeting and the consensus was that if we keep the balance of Armory-branded videos at around 50% of the content, this addition is a net win for the Spinnaker site. 

@demobox I plan to reach out to you directly on Spinnaker Slack after creating this, but TLDR I'm hoping that adding this to the "success stories" page is analogous enough to the "Who's Using Spinnaker" page you referenced in your comment on pull 1073, and that a better balance of Armory interviews vs. conference presentations and videos from other sponsors will provide a more accurate view of the project.

This implementation is simple and could be built upon in the future (I would love to see or create a gridded video area in a future PR for example). Another future item is a link to the Spinnaker youtube channel. Before adding that though, I'd like to follow up on maintaining the Spinnaker channel; I'm hoping to gain access to it so I can add more recent videos and plan to reach out to @aglover to inquire about that. One action item after getting access would be for me to migrate videos from the Armory account to the Spinnaker account so they can be featured on this page without the Armory shield branding in the top-left.

One technical note: the hack I used to push the Armory sail bottom image down so content wouldn't overlap it was a bunch of <br> tags. If any objections or suggestions for a better solution, happy to entertain!

Thanks for considering!!!

### CODE REVIEW

**Comment by dnilasor** [comment]
OK All FREUDIAN SLIP it's a SPINNAKER SAIL not an Armory sail please forgive me : - )

File: success-stories/index.html [hunk 1]
```diff
@@ -53,6 +53,25 @@ <h3>Testimonials</h3>
...
     </p>
-    
-    <img class="bottom-right-swoosh" src="{{ "assets/images/bottom-right-swoosh.svg" | absolute_url }}" alt="Spinnaker Swoosh"/>
 </div>
+<div class="videos-container">
+    <h3>Videos</h3>
+    <p>Conference talks, interviews, and video testimonials from the Spinnaker Community</p>
+    <div class="videos-wrapper">
+      <h4>Jens Vanderhaeghe of Airbnb on Migrating From Custom Deployment Tooling To Spinnaker</h4>
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/vQCkAkya_Yg" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```
Comment (317brian) [inline success-stories/index.html]: Could we shrink the iframe? They videos seem needlessly gigantic on the page

Comment (dnilasor) [reply success-stories/index.html]: Sure. I agree they look gigantic and several times have checked that they're the same size as the tutorial videos. I think it's just the different formatting on that md page that makes those look less gross? Working on shrinkage now

**Comment by demobox** [comment]
@dnilasor Thanks for the heads-up! I've added @dorbin as a reviewer and will sync up with him regarding the points you raised in your explanation

**Comment by aglover** [comment]
LGTM

**Review by dorbin — APPROVED** [review]

**Comment by 317brian** [comment]
Thanks for taking a look too @dorbin. I chatted with @dnilasor, and she's going to try to fix the giant videos (adjusting the iframe size isn't doing it). If we can't get it figured out, I think we should merge it Thursday before Spinnaker Summit.

**Comment by dnilasor** [comment]
Thanks @dorbin and @brian-armory! just pushed some CSS updates. It ain't beautiful but ready for the web and 2.0 will be even better!

**Comment by 317brian** [comment]
Thank you for making them not gigantic! It should probably be a grid, but I'll file a separate issue for that.

**Pull request merged**